packages/launchpad/src/components/topnav/TopNav.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <TopNavList v-if="versionList">
+    <template #heading="{ open }">
+      <i-cy-box_x16
+        class="group-hocus:icon-dark-indigo-500 group-hocus:icon-light-indigo-50 h-16px w-16px"
+        :class="open ? 'icon-dark-indigo-500 icon-light-indigo-50' : 'icon-dark-gray-500 icon-light-gray-100'"
+      />
+      <span>v{{ versionList[0].version }}</span>
+    </template>
+    <TopNavListItem
+      v-for="(item, index) in versionList"
+      :key="item.version"
+      :selectable="!!index"
```

Comment:
We're using the index really heavily to say "currentVersion", but it's a bit awkward, and maybe sometimes the current version isn't the latest version. Could we add a "current" property to the version list and use that instead?

Reply:
In this case it really should be the first item in the list that is active. If the latest version is not currently installed, the UI for the open state is different:

<img width="295" alt="Screen Shot 2021-10-04 at 7 53 53 AM" src="https://user-images.githubusercontent.com/8340719/135847436-4e9c56e5-b3cf-4c9f-bec7-5c2f0c79967f.png">

This is something that came up last week with Ryan when trying to solve for "what if the installed version is like 6 releases behind?".

I think what will probably happen in https://cypress-io.atlassian.net/browse/UNIFY-382 is:

- get the content for this list out of gql (last 4 cy releases + currently installed version, and a way to know the released date of the currently installed version no matter how old)
- if currently installed release version doesn't match the latest release version, we show the "upgrade" state
- if they do match, we are up to date and it's always the first item in the list

Reply:
Oh true, true.